jsonrpc/filter_manager_test.go

```diff
@@ -247,6 +247,10 @@ func (m *mockStore) SetAccount(addr types.Address, account *state.Account) {
 	m.accounts[addr] = account
 }
 
+func (m *mockStore) GetCapacity() (uint64, uint64) {
+	return 0, 0
```

Comment:
Is there any way to have this data be updated as the mockStore is being updated?

Reply:
What do you think that would accomplish? @zivkovicmilos